### PR TITLE
demjson

### PR DESCRIPTION
**Pull request by scopatz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
This is a noarch python recipe ready for review @conda-forge/staged-recipes

File: recipes/demjson/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+test:
+  imports:
+    - demjson
+
+about:
+  home: http://deron.meranda.us/python/demjson/
+  license: LGPL-3.0-or-later
+  license_family: GPL
+  license_file: LICENSE.txt
+  summary: 'Provides methods for encoding and decoding JSON formatted data'
```
Comment (synapticarbors) [inline recipes/demjson/meta.yaml:33]: maybe add `dev_url: https://github.com/dmeranda/demjson`. Also note that the `home` url seems to be dead, even though that seems to still be the official one.

Comment (scopatz) [reply recipes/demjson/meta.yaml:33]: Yeah, I realized that about the home URL.  It is probably what they want though

**Comment by scopatz** [comment]
Ready for review again @conda-forge/staged-recipes!

**Comment by scopatz** [comment]
ping @conda-forge/staged-recipes

**Pull request merged**